lib/Scvmm/miq_winrm.rb

```diff
@@ -0,0 +1,53 @@
+
+require 'winrm'
```

Comment:
Should this file live in a more generic location like lib/util?  I don't think it's SCVMM specific.